docs/_pages/0110-configuration.md

```diff
@@ -264,10 +264,46 @@ If the environment variable is not set, the default buffer size is `10`.
 cryptographic key input for **VSecM Safe**, instead of letting the bootstrap
 flow automatically compute cryptographic keys.
 
-If the environment variable is not set or its value is not `"true"`, the bootstrap
-flow will compute the cryptographic keys manually. If this variable is set to
-`"true"` then a human operator has to provide the necessary cryptographic keys
-using **VSecM Sentinel**.
+If the environment variable is not set or its value is not `"true"`, the 
+bootstrap flow will **not** compute the cryptographic keys automatically. 
+
+If this variable is set to `"true"` then a human operator has to provide the 
+necessary cryptographic keys using **VSecM Sentinel** for **VSecM** Safe** to
+unlock itself and start serving API requests.
+
+> **Setting the Master Key Manually**
+>
+> You can [set the master key programmatically using 
+> **VSecM Sentinel**][vsecm-sentinel-master]. 
+{: .block-tip}
+
+[vsecm-sentinel-master]: https://vsecm.com/docs/cli/
+
+The control offered by this approach changes the threat boundary of 
+**VSecM Safe**. With this approach, the responsibility of securing the
+master key is on you as the operator. This is different than the default 
+behavior of **VSecM Safe** where the master key is randomly-generated in a 
+cryptographically secure way and stored in a Kubernetes `Secret`.
+
+Using a Kubernetes `Secret` to store the master key is still secure, 
+especially if you encrypt your `etcd` and establish a **tight RBAC** over the 
+Kubernetes `Secret`that stores the master key.
+
+> **Master Key Storage in Manual Input Mode**
+> 
+> As of *v0.21.5* of **VMware Secrets Manager**, the master key is not stored 
+> in a Kubernetes `Secret` if you use the manual key input approach. This 
+> behavior is subject to change in the future, depending on a configuration 
+> environment variable, the master key might be stored in a Kubernetes `Secret` 
+> even if you use the manual key input approach.
+> 
+> However, this means that if you use the manual key input approach, you will
+> have to re-enter the master key every time you restart **VSecM Safe** or
+> every time the pod is evicted by the scheduler.
```

Comment:
There is some work to do here… once that’s done, we might want to either remove or change this tip.